### DISCUSSION TITLE
Album artwork not showing up.

### DISCUSSION THREAD

**Discussion by s0hv** [s0hv/foo_discord_rich#2] [Q&A]
Copied this here as it's a more fitting place for discussions like this.
By @Gallus780

> I've gone through all steps, even updated foobar2000 with all its plugins. I have downloaded Pyhton both from its official website and Microsoft Store. However the cover art still doesn't show in Discord. What is wrong?

Try opening the console from View -> Console and see what kind of messages pop up when starting a new track. It should give some clue as to what is going wrong.

**Comment by blkwald** [comment]
i have the same issue, but i can't figure out what goes wrong 
this is what the console reads when playing a new track :

> --> populate Smoothplaylist with PlayingPlaylist call_id:10 Parent panel:MainPanel
> Opening track for playback: "F:\Music\My Bloody Valentine\1991 -  Loveless (flac)\05 - When You Sleep.flac"
> 726469593
> Biography Server: allmusic review: Loveless [1st edition Japan COCY-9243] / My Bloody Valentine: not found
> Biography Server: wikipedia: Loveless [1st edition Japan COCY-9243]: not found

**Reply by s0hv** [reply]
It seems like the album artwork feature is not enabled. Can you make sure you have followed the setup steps as described here and report if that helped or not https://s0hv.github.io/foo_discord_rich/docs/album-artwork/

**Reply by HaizeSteiner** [reply]
Discord Rich Presence Integration v1.3.0: Failed to run command '"C:\Program Files\foobar2000\foobar2000-catbox.exe"'

I don't know why but it was working fine a month ago and now it's giving me issues and not displaying the art cover.

i tried again with python with numerous scripts and gave me the following error: Discord Rich Presence Integration v1.3.0: artwork uploader exited with status: 1 and error: Traceback (most recent call last):
  File "C:\Program Files\foobar2000\foobar_catbox_album_art_script_experimental.py", line 5, in <module>
    import requests

**Reply by s0hv** [reply]
@HaizeSteiner I have not built foobar2000-catbox.exe so I don't know why it does not work. As for the python script, it looks like requests is not installed.

**Reply by HaizeSteiner** [reply]
Still gives me the same problem with the imgur. I followed the tutorial and the guide on your website. It used to work until this morning now gives me Discord Rich Presence Integration v1.3.0: Failed to run command "C:\Program Files\foobar2000\imgur-uploader.exe"

Maybe due to the x64 architecture? But always used the same settings i didn't touch anything besides the directories to fix the problem.
I already installed the python dependencies.

**Reply by s0hv** [reply]
If it worked until this morning it was probably the doing of windows and that's out of my control
